src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Pre-built dasboards
+tags:
+  - Query your data
+  - Explore and query data
+  - Dashboards
+  - Pre-built dashboards
+---
+
+import dashboardsDashboardIndex from 'images/dashboards_screenshot-full_dashboard-index.png'
+
+import dashboardsPrebuiltViewAll from 'images/dashboards_screenshot-full_prebuilt-view-all.png'
+
+import dashboardsPrebuiltSearch from 'images/dashboards_screenshot-full_prebuilt-search.png'
+
+
+Through the New Relic UI, you can access a collection of commonly pre-built dashboards, to help you instantly visualize your data. These pre-built dashboards enhance quickstart dashboard discovery, speed up the installation process, and get to instantly visualize your data.
+
```

Comment:
@nbaenam this is looking much better! Two notes:

1. "commonly pre-built" -> "commonly used pre-built" OR "popular pre-built"
2.  The second sentence is not really necessary.

Reply:
Agreed- maybe something like:

> Through the New Relic UI, you can access a collection of popular pre-built dashboards, to help you instantly visualize your data for common use cases.

Reply:
Done!